sailthru/Sailthru_Client.php

```diff
@@ -1563,4 +1581,49 @@ protected function prepareJsonPayload(array $data, array $binary_data = array())
         }
         return $payload;
     }
+
+    /**
+     * get the rate limit information for the very last call with given action and method
+     * @param string $action
+     * @param string $method GET, POST or DELETE
+     * @return array or null
+     */
+    public function getLastRateLimitInfo($action, $method) {
```

Comment:
feel like it's more clear to use *RateLimit* to represent the feature. *rate* itself might be ambiguous. What do you think @georgekliao?